openapi/openapi.yaml

```
@@ -1361,6 +1361,8 @@ paths:
     $ref: paths/storefront/orders@{id}@pause.yaml
   '/storefront/subscriptions/{id}/change-items':
     $ref: paths/storefront/subscriptions@{id}@change-items.yaml
+  '/storefront/subscriptions/{id}/reactivations':
```

This is a subresource. Does the reactivation have any ID? Kind of -- it's a quote ID.
```suggestion
  '/storefront/quotes/{id}/':
```

Or `/storefront/reactivations/{id}/`